extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class TriviaQuizzesHooks extends WikiaController {
+	public static function onBeforePageDisplay( OutputPage $out ) {
+		global $wgTitle, $wgEnableTriviaQuizzesExt, $wgTriviaQuizzesEnabledPages;
+
+		if ( $wgEnableTriviaQuizzesExt && in_array( $wgTitle->getPrefixedText(), $wgTriviaQuizzesEnabledPages ) ) {
```

Comment:
```suggestion
		if ( $wgEnableTriviaQuizzesExt && in_array( $out->getTitle()->getPrefixedText(), $wgTriviaQuizzesEnabledPages ) ) {
```

And as a result remove `$wgTitle` global declaration above.

Reply:
done in b28949d